bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-0/Radion_VBF_ZZ_inclu/Radion_VBF_ZZ_inclu_narrow_M1400/Radion_VBF_ZZ_inclu_narrow_M1400_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model heft_radion
+define p = u c d s u~ c~ d~ s~
+define j = p
+define ferm = u c d s b u~ c~ d~ s~ b~ e- mu- ta- e+ mu+ ta+ vl vl~
```

Comment:
Hi Eiko,
When you say "For b quarks that come from decays of particles, they are still kept."
You mean using syntax like this?